src/factom-event-emitter.js

```diff
@@ -321,6 +437,28 @@ class FactomEventEmitter extends EventEmitter {
             }
         });
     }
+
+    // Emit new pending factoid transactions for user-defined FCT address
+    _emitPendingFactoidTransaction(fctAddress, transactions) {
```

Comment:
This emits a single transaction now:
transactions => transaction
(in the comment and in the variable name)